src/utils/parseWSManResponseBody.test.ts

```diff
@@ -0,0 +1,76 @@
+    const response = parseBody(xmlResponse)
+    expect(response).toEqual(generalSettings)
+  })
+
+  it('Should fail and return null when a message does not contain \r\n', async () => {
+    const xmlResponse: HttpZResponseModel = {
+      protocolVersion: 'HTTP/1.1',
+      statusCode: 200,
+      statusMessage: 'OK',
+      headersSize: 218,
+      bodySize: 2165,
+      headers: [
+        { name: 'Date', value: 'Mon, 24 Jan 2022 13:28:09 GMT' },
+        {
+          name: 'Server',
+          value: 'Intel(R) Active Management Technology 15.0.23.1706'
+        },
+        { name: 'X-Frame-Options', value: 'DENY' },
+        {
+          name: 'Content-Type',
+          value: 'application/soap+xml; charset=UTF-8'
+        },
+        { name: 'Transfer-Encoding', value: 'chunked' }
+      ],
+      body: {
+        contentType: 'application/soap+xml',
+        boundary: '',
+        params: [],
+        text: '<?xml version="1.0" encoding="UTF-8"?><a:Envelope xmlns:a="http://www.w3.org/2003/05/soap-envelope" xmlns:b="http://schemas.xmlsoap.org/ws/2004/08/addressing" xmlns:c="http://schemas.dmtf.org/wbem/wsman/1/wsman.xsd" xmlns:d="http://schemas.xmlsoap.org/ws/2005/02/trust" xmlns:e="http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-secext-1.0.xsd" xmlns:f="http://schemas.dmtf.org/wbem/wsman/1/cimbinding.xsd" xmlns:g="http://intel.com/wbem/wscim/1/amt-schema/1/AMT_GeneralSettings" xmlns:xsi="http://www.w3.org/2001/XMLSchema-ins'
+      }
+    }
+    const response = parseBody(xmlResponse)
+    expect(response).toEqual(undefined)
```

Comment:
Should use toBeUndefined() instead of toEqual

Reply:
done